openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -253,7 +258,9 @@ def get_course_cohorts(course, assignment_type=None):
     # Migrate cohort settings for this course
     migrate_cohort_settings(course)
 
-    query_set = CourseUserGroup.objects.filter(
+    query_set = CourseUserGroup.objects.exclude(
+        name=CourseUserGroup.INTERNAL_NAME
+    ).filter(
```

Comment:
I'll expand on `CourseUserGroup.INTERNAL_NAME` below, these lines are to exclude it from appearing in the list of user-visible cohorts.